### DISCUSSION TITLE
How can I remove image preview of uploaded files.

### DISCUSSION THREAD

**Discussion by d3osaju** [ant-design/ant-design#37813] [Q&A] [answered]
I am uploading multiple file using custom request and upload button.
But I can't clear the preview of the images after image is uploaded 

How can I clear all the file preview images at once

**Comment by NadiaKishchuk** [comment]
Did you resolve this problem? I have the same one.

**Reply by d3osaju** [reply]
Yes , I think so .
But don't remember how i done it !

**Reply by martinstefanovic** [reply]
@Deosaju then why are you answering here at all if you are not going to tell how you solved the problem....

**Reply by MaymoonaAlBoloshi** [reply]
🤣 @Deosaju the answer I feared all my life is here

**Reply by linxianxi** [reply, marked as answer]
try `showUploadList` or `onPreview={() => {}}`

**Reply by jaskaran-delcu** [reply]
set file list to empty array

fileList={[]}

**Reply by an-pham916** [reply]
Thank you!
